syncd/RedisClient.cpp

```diff
@@ -231,6 +231,32 @@ void RedisClient::setDummyAsicStateObject(
     m_dbAsic->hset(strKey, "NULL", "NULL");
 }
 
+void RedisClient::setDummyAsicStateObject(
+        _In_ sai_object_id_t* objectVid,
```

Comment:
name this objectVids, and it probably be better if  this would be const std::vector<sai_object_id_t>

Reply:
done